deployment/cdk/opensearch-service-migration/lib/service-stacks/traffic-replayer-stack.ts

```diff
@@ -89,10 +89,11 @@ export class TrafficReplayerStack extends MigrationServiceCore {
                 "secretsmanager:DescribeSecret"
             ]
         })
+        const openSearchPolicy = createOpenSearchIAMAccessPolicy(<string>props.env?.region, <string>props.env?.account)
+        const openSearchServerlessPolicy = createOpenSearchServerlessIAMAccessPolicy(<string>props.env?.region, <string>props.env?.account)
 
         const deployId = props.addOnMigrationDeployId ? props.addOnMigrationDeployId : props.defaultDeployId
-        const cdkDomainEndpoint = StringParameter.valueForStringParameter(this, `/migration/${props.stage}/${deployId}/osClusterEndpoint`)
-        const osClusterEndpoint = props.customTargetEndpoint ? props.customTargetEndpoint : `https://${cdkDomainEndpoint}:443`
+        const osClusterEndpoint = StringParameter.valueForStringParameter(this, `/migration/${props.stage}/${deployId}/osClusterEndpoint`)
```

Comment:
This can be ignored, but given that some clusters might be OpenSearch and others might be Elasticsearch, even if both of these are being managed by the OpenSearch managed service, would it be better to drop the 'os' from the prefix?  I concede that even if it is, now might not be the right time to make the change.